dipy/reconst/recspeed.pyx

```diff
@@ -57,6 +58,8 @@ def remove_similar_vertices(
     return_indices : {False, True}, optional
         If True, return `indices` as well as `vertices` and maybe `mapping`
         (see below).
+    is_symmetric : {False, True}, optional
+        If False, v is considered different from -v.
 
```

Comment:
I suggest changing to `If True, v is considered equal to -v.`

Reply:
Could you also edit lines 45-46 and remove the sentence "Vertex v and -v are considered the same so if v and -v are both in  `vertices` only one is kept."